internal/js_library/v2/js_library.bzl

```diff
@@ -0,0 +1,172 @@
+
+"""js_library allows defining a set of javascript sources to be used with ts_devserver"""
```

Comment:
It's really about writing ESnext Javascript while running in runtimes of today

The fact that it's made to interop with `ts_devserver` is only interesting if you also author typescript code